helm/templates/processbot.yaml

```diff
@@ -0,0 +1,85 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: parity-processbot
+  namespace: {{ .Values.app.KUBE_NAMESPACE }}
+  labels:
+    app: parity-processbot
+spec:
+  ports:
+    - name: backend
+      port: {{ .Values.app.WEBHOOK_PORT }}
+  selector:
+    app: parity-processbot
+---
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: parity-processbot
+  namespace: {{ .Values.app.KUBE_NAMESPACE }}
+  labels:
+    app: parity-processbot
+spec:
+  selector:
+    matchLabels:
+      app: parity-processbot
+  serviceName: parity-processbot
+  updateStrategy:
+    type: RollingUpdate
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: parity-processbot
+    spec:
+      volumes:
+      - name: processbot-key-volume
+        secret:
+          secretName: processbot-key
+          defaultMode: 256
+      - name: database
+        persistentVolumeClaim:
+          claimName: processbot-pv-claim
+      containers:
+      - name: parity-processbot
+        imagePullPolicy: Always
+        image: paritytech/processbot:{{ .Values.app.DOCKER_TAG }}
+        volumeMounts:
+        - name: processbot-key-volume
+          mountPath: "/etc/processbot-key"
+          readOnly: true
+        - name: database
+          mountPath: /usr/local/share/db
+        ports:
+        - name: backend
+          containerPort: {{ .Values.app.WEBHOOK_PORT }}
+        env:
+            - name: RUST_BACKTRACE
+              value: full
+            - name: RUST_LOG
+              value: debug
+            - name: INSTALLATION_LOGIN
+              value: {{ .Values.app.INSTALLATION_LOGIN }}
+            - name: PRIVATE_KEY_PATH
+              value: "/etc/processbot-key/PROCESSBOT_KEY"
+            - name: WEBHOOK_PORT
+              value: {{ quote .Values.app.WEBHOOK_PORT }}
+            - name: WEBHOOK_SECRET
+              value: {{ .Values.app.WEBHOOK_SECRET }}
```

Comment:
@lovelaced The previous version of the tags are using secretKeyRef:

https://github.com/paritytech/parity-processbot/blob/bee58c73fa2a751dc8d0be3b3b1b87cf2b12d1f1/kubernetes/processbot/templates/processbot.yaml#L70

Here I am using the secret directly:

https://github.com/paritytech/parity-processbot/blob/a228e3ea25e9361e2a6cc521e526bdd56ae71ded/helm/templates/processbot.yaml#L69

do you prefer either approach?